### PR TITLE
timeseries: fix runs table scrollable issue

### LINKED ISSUE #5019: Last runs can not be scrolled on screen in time series plugin
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=8, micro=5, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='studentPC', release='5.8.0-50-generic', version='#56~20.04.1-Ubuntu SMP Mon Apr 12 21:46:35 UTC 2021', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==2.5.0
INFO: installed: tensorflow==2.5.0
INFO: installed: tensorflow-estimator==2.5.0
INFO: installed: tensorboard-data-server==0.6.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.5.0'

--- check: tensorflow_python_version
2021-05-27 09:02:54.500209: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-05-27 09:02:54.500227: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
INFO: tensorflow.__version__: '2.5.0'
INFO: tensorflow.__git_version__: 'v2.5.0-rc3-213-ga4dfb8d1a71'

--- check: tensorboard_data_server_version
INFO: data server binary: '/home/student/anaconda3/envs/cut_in/lib/python3.8/site-packages/tensorboard_data_server/bin/server'
INFO: data server binary version: b'rustboard 0.6.1'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/home/student/anaconda3/envs/cut_in/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'studentPC'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=14295060, st_dev=66306, st_nlink=2, st_uid=1000, st_gid=1000, st_size=4096, st_atime=1619509692, st_mtime=1621934199, st_ctime=1621934199)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/home/student/anaconda3/envs/cut_in/lib/python3.8/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py @ file:///tmp/build/80754af9/absl-py_1603893106258/work
altair==4.1.0
argon2-cffi @ file:///tmp/build/80754af9/argon2-cffi_1596828493937/work
astor==0.8.1
astroid @ file:///tmp/build/80754af9/astroid_1592495912941/work
astunparse==1.6.3
async-generator==1.10
attrs @ file:///tmp/build/80754af9/attrs_1604765588209/work
backcall==0.2.0
base58==2.0.1
bleach @ file:///tmp/build/80754af9/bleach_1600439572647/work
blinker==1.4
boto3==1.16.28
botocore==1.19.28
Brotli==1.0.9
brotlipy==0.7.0
cachetools @ file:///tmp/build/80754af9/cachetools_1596822027882/work
certifi==2020.12.5
cffi @ file:///tmp/build/80754af9/cffi_1606255081583/work
chardet @ file:///tmp/build/80754af9/chardet_1605303185383/work
click==7.1.2
configparser==5.0.1
cryptography @ file:///tmp/build/80754af9/cryptography_1605544487601/work
cycler==0.10.0
dash==1.19.0
dash-bootstrap-components==0.11.3
dash-core-components==1.15.0
dash-html-components==1.1.2
dash-renderer==1.9.0
dash-table==4.11.2
dataclasses==0.6
decorator==4.4.2
defusedxml==0.6.0
dill==0.3.3
docker-pycreds==0.4.0
elasticsearch @ file:///tmp/build/80754af9/elasticsearch_1606082901114/work
entrypoints==0.3
enum-compat==0.0.3
Flask==1.1.2
Flask-Compress==1.9.0
flatbuffers==1.12
future==0.18.2
gast==0.4.0
gitdb==4.0.5
GitPython @ file:///tmp/build/80754af9/gitpython_1603928675344/work
google-auth @ file:///tmp/build/80754af9/google-auth_1600960338579/work
google-auth-oauthlib @ file:///tmp/build/80754af9/google-auth-oauthlib_1603929124518/work
google-pasta==0.2.0
grpcio==1.34.1
h5py==3.1.0
idna @ file:///tmp/build/80754af9/idna_1593446292537/work
importlib-metadata @ file:///tmp/build/80754af9/importlib-metadata_1602276842396/work
iniconfig==1.1.1
ipykernel @ file:///tmp/build/80754af9/ipykernel_1596207638929/work/dist/ipykernel-5.3.4-py3-none-any.whl
ipython @ file:///tmp/build/80754af9/ipython_1604101197014/work
ipython-genutils @ file:///tmp/build/80754af9/ipython_genutils_1606773439826/work
ipywidgets @ file:///tmp/build/80754af9/ipywidgets_1601490159889/work
isort @ file:///tmp/build/80754af9/isort_1602603989581/work
itsdangerous==1.1.0
jedi @ file:///tmp/build/80754af9/jedi_1598371611696/work
Jinja2==2.11.2
jmespath==0.10.0
joblib @ file:///tmp/build/80754af9/joblib_1601912903842/work
jsonschema @ file:///tmp/build/80754af9/jsonschema_1602607155483/work
jupyter-client @ file:///tmp/build/80754af9/jupyter_client_1601311786391/work
jupyter-core @ file:///tmp/build/80754af9/jupyter_core_1606148996965/work
jupyterlab-pygments @ file:///tmp/build/80754af9/jupyterlab_pygments_1601490720602/work
keras-nightly==2.5.0.dev2021032900
Keras-Preprocessing==1.1.2
kiwisolver @ file:///tmp/build/80754af9/kiwisolver_1604014535162/work
lazy-object-proxy==1.4.3
Mako==1.1.3
Markdown @ file:///tmp/build/80754af9/markdown_1605111056890/work
MarkupSafe==1.1.1
matplotlib @ file:///tmp/build/80754af9/matplotlib-base_1603378225747/work
mccabe==0.6.1
mistune==0.8.4
mkl-fft==1.2.0
mkl-random==1.1.1
mkl-service==2.3.0
multiprocess==0.70.11.1
nbclient @ file:///tmp/build/80754af9/nbclient_1602783176460/work
nbconvert @ file:///tmp/build/80754af9/nbconvert_1601914830498/work
nbformat @ file:///tmp/build/80754af9/nbformat_1602783287752/work
nest-asyncio @ file:///tmp/build/80754af9/nest-asyncio_1606153767164/work
notebook @ file:///tmp/build/80754af9/notebook_1601501575118/work
numpy @ file:///tmp/build/80754af9/numpy_and_numpy_base_1603570489231/work
oauthlib==3.1.0
olefile==0.46
opencv-python==4.4.0.46
opt-einsum==3.3.0
p-tqdm==1.3.3
packaging==20.4
pandas @ file:///tmp/build/80754af9/pandas_1602088120436/work
pandocfilters @ file:///tmp/build/80754af9/pandocfilters_1605120460739/work
parso==0.7.0
pathos==0.2.7
pathtools==0.1.2
pdoc3 @ file:///tmp/build/80754af9/pdoc3_1606517329097/work
pexpect @ file:///tmp/build/80754af9/pexpect_1605563209008/work
pickleshare==0.7.5
Pillow @ file:///tmp/build/80754af9/pillow_1603822255246/work
pip==20.3
plotly==4.13.0
pluggy==0.13.1
pox==0.2.9
ppft==1.6.6.3
prometheus-client @ file:///tmp/build/80754af9/prometheus_client_1606344362066/work
promise==2.3
prompt-toolkit @ file:///tmp/build/80754af9/prompt-toolkit_1602688806899/work
protobuf==3.15.6
psutil==5.8.0
ptyprocess @ file:///tmp/build/80754af9/ptyprocess_1605560620615/work/dist/ptyprocess-0.6.0-py2.py3-none-any.whl
py==1.10.0
pyarrow==0.15.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser @ file:///tmp/build/80754af9/pycparser_1594388511720/work
pydeck==0.5.0
Pygments @ file:///tmp/build/80754af9/pygments_1604103097372/work
PyJWT==1.7.1
pylint @ file:///tmp/build/80754af9/pylint_1598623985952/work
pyOpenSSL @ file:///tmp/build/80754af9/pyopenssl_1606517880428/work
pyparsing==2.4.7
pyrsistent @ file:///tmp/build/80754af9/pyrsistent_1600141720057/work
PySocks @ file:///tmp/build/80754af9/pysocks_1605305779399/work
pytest==6.2.1
python-dateutil==2.8.1
pytorch-warmup==0.0.4
pytz @ file:///tmp/build/80754af9/pytz_1606604771399/work
PyYAML==5.3.1
pyzmq==20.0.0
requests @ file:///tmp/build/80754af9/requests_1606691187061/work
requests-oauthlib==1.3.0
retrying==1.3.3
rope @ file:///tmp/build/80754af9/rope_1602264064449/work
rsa @ file:///tmp/build/80754af9/rsa_1596998415516/work
ruptures==1.1.1
s3transfer==0.3.3
scipy @ file:///tmp/build/80754af9/scipy_1597686649129/work
Send2Trash==1.5.0
sentry-sdk==0.19.5
setuptools==50.3.1.post20201107
shortuuid==1.0.1
sip==4.19.13
six @ file:///tmp/build/80754af9/six_1605205327372/work
smmap @ file:///tmp/build/80754af9/smmap_1597003226630/work
stqdm==0.0.3
streamlit==0.80.0
subprocess32==3.5.4
tensorboard==2.5.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.6.0
tensorflow==2.5.0
tensorflow-estimator==2.5.0
termcolor==1.1.0
terminado==0.9.1
terminaltables==3.1.0
testpath==0.4.4
toml @ file:///tmp/build/80754af9/toml_1592853716807/work
toolz==0.11.1
torch==1.7.0
torchvision==0.8.1
tornado==6.0.4
tqdm==4.60.0
traitlets @ file:///tmp/build/80754af9/traitlets_1602787416690/work
typing-extensions @ file:///tmp/build/80754af9/typing_extensions_1598376058250/work
tzlocal==2.1
urllib3 @ file:///tmp/build/80754af9/urllib3_1603305693037/work
validators==0.18.1
wandb==0.10.18
watchdog==0.10.4
wcwidth @ file:///tmp/build/80754af9/wcwidth_1593447189090/work
webencodings==0.5.1
Werkzeug==1.0.1
wheel==0.35.1
widgetsnbextension==3.5.1
wrapt==1.12.1
zipp @ file:///tmp/build/80754af9/zipp_1604001098328/work

``````

</details>

Tested Browsers: 
1. Firefox, 88.0.1 (64-bit), Mozilla Firefox for Ubuntu canonical - 1.0
2. Chrome, Version 90.0.4430.212 (Official Build) (64-bit)

Screenshot of issue:
last_run_not_visible

## Issue description

When there are too many runs, the last run can not be selected in the time series plugin. It can not be scrolled onto the screen, since the scrolling area is partially outside the browser window. 

This is not an issue in the standard "scalar" tab of tensorboard. In that tab I am able to see run 00381 (from my screenshot) without a problem.

Even though tensorflow is installed in my virtualenv (because I needed it at some point) I am using tensorboard with PyTorch only. However, I don't think this affects the behavior described in the issue at all, because it should only affect how the metrics are logged.

Thank you :)

### PR DESCRIPTION
**Pull request by stephanwlee**
When a `<runs-table-component>` has height = 100px, the .table-container
also had height = 100px which is not what we intended. We intended it to
have 100px - heightOfFilterRow. We now use the proper flex syntax for
the proper layout.

This purely visual bug is hard to test in unit test and will be covered
via screenshot tests internally.

Fixes #5019.

### CODE REVIEW

**Review by psybuzz — APPROVED** [review]
Nice quick fix
